tests/common/test_utils.py

```diff
@@ -18,6 +18,17 @@
 from model_analyzer.perf_analyzer.perf_config import PerfAnalyzerConfig
 
 
+def create_yaml(string):
```

Comment:
This method isnt creating anything related to yaml. It is converting a string to bytes. 
Can we rename it to something like convert_to_bytes()?

Reply:
Done!